packages/e2e-tests/specs/editor/various/taxonomies.test.js

```diff
@@ -141,8 +139,10 @@ describe( 'Taxonomies', () => {
 		// Click the tag input field.
 		await tagInput.click();
 
+		const tagName = 'tag-' + random( 1, Number.MAX_SAFE_INTEGER );
```

Comment:
@youknowriad - this is the test which pollutes the database. We probably should improve the logic which cleans up the database by removing tags and categories in addition to posts and pages.

Reply:
yeah, we can also add an `afterAll` step in this test to clean things.